src/scripts/main.js

```diff
@@ -1,3 +1,91 @@
 'use strict';
 
 // write code here
+const container = document.querySelector('.container');
+const table = document.querySelector('tbody');
+const rowsCount = table.rows;
+const columnsCount = table.rows[0].cells;
+const appendRowBtn = document.querySelector('.append-row');
+const removeRowBtn = document.querySelector('.remove-row');
+const appendColumnBtn = document.querySelector('.append-column');
+const removeColumnBtn = document.querySelector('.remove-column');
+
+function addRow() {
+  const count = [...table.rows[0].cells];
+  const tr = document.createElement('tr');
+
+  count.forEach(c => {
+    const td = document.createElement('td');
+
+    tr.append(td);
+    table.append(tr);
+  });
+}
+
+function addColumn() {
+  const count = [...table.rows];
+
+  count.forEach(row => {
+    const td = document.createElement('td');
+
+    row.append(td);
+  });
+}
+
+function removeRow() {
+  table.lastElementChild.remove();
+}
+
+function removeColumn() {
+  const count = [...table.rows];
+
+  count.forEach(row => {
+    const lastElement = row.lastElementChild;
+
+    return lastElement.remove();
+  });
+}
+
+function addRemove(buttonClass, button) {
+  switch (buttonClass) {
+    case 'append-row':
+      addRow();
+
+      rowsCount.length >= 10
+        ? button.setAttribute('disabled', '')
+        : removeRowBtn.disabled = false;
+      break;
+    case 'remove-row':
+      removeRow();
+
+      rowsCount.length <= 2
```

Comment:
Instead of using the ternary operator to set the 'disabled' attribute, use a standard if-else statement for better readability. Additionally, it's safer to use button.disabled = true/false to disable/enable the button.